packages/taraxa-sdk/.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  overrides: [
+    {
+      // JavaScript and JSX
+      files: ['*.{ts,tsx}'],
```

Comment:
```suggestion
      files: ['*.{ts}'],
```
As far as I can see there's no tsx inside